linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/H2LoggerConfig.scala

```diff
@@ -0,0 +1,56 @@
+package io.buoyant.linkerd.protocol.h2
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.buoyant.h2.{Request, Response}
+import com.twitter.finagle.{Filter, ServiceFactory, Stack, Stackable}
+import com.twitter.finagle.stack.nilStack
+import io.buoyant.config.PolymorphicConfig
+
+abstract class H2LoggerConfig extends PolymorphicConfig { config =>
```

Comment:
`HttpLoggerConfig`/`H2LoggerConfig`: `new Stack.Module` itself already does good job of reducing to the essentials.

`IstioLoggerConfig`: I could pull out host, port, client, etc into it's own trait (`IstioLoggerConfig extends HttpConfig with IstioLoggerConfigBase`), but having overrides in all three places was unnecessarily confusing. 


In general I only focus on DRYing things up when there are 3 copies of something, and didn't find a lot of value in sharing code between configs.